packages/x-date-pickers/src/internals/components/PickerProvider.tsx

```diff
@@ -202,12 +228,87 @@ export interface PickerActionsContextValue<
   TValue extends PickerValidValue,
   TView extends DateOrTimeViewWithMeridiem,
   TError = string | null,
-> extends UsePickerValueActionsContextValue<TValue, TError>,
-    UsePickerViewsActionsContextValue<TView> {}
+> {
+  /**
+   * Set the current value of the picker.
+   * @param {TValue} value The new value of the picker.
+   * @param {SetValueActionOptions<TError>} options The options to customize the behavior of this update.
+   */
+  setValue: (value: TValue, options?: SetValueActionOptions<TError>) => void;
+  /**
+   * Set the current open state of the Picker.
+   * ```ts
+   * setOpen(true); // Opens the picker.
+   * setOpen(false); // Closes the picker.
+   * setOpen((prevOpen) => !prevOpen); // Toggles the open state.
+   * ```
+   * @param {React.SetStateAction<boolean>} action The new open state of the Picker.
+   * It can be a function that will receive the current open state.
```

Comment:
```suggestion
   * Set the current open state of the Picker.
   * It can be a function that will receive the current open state as parameter.
   * ```ts
   * setOpen(true); // Opens the picker.
   * setOpen(false); // Closes the picker.
   * setOpen((prevOpen) => !prevOpen); // Toggles the open state.
   * ```
   * @param {React.SetStateAction<boolean>} action The new open state of the Picker.
```